solutions/usaco-941.mdx

```diff
@@ -107,72 +107,47 @@ print('yes')
 <JavaSection>
 
 ```java
-import java.util.*;
 import java.io.*;
+import java.util.*;
+
+public class Evolution {
+	static boolean[][] hasChar = new boolean[25][125];
```

Comment:
put 25 and 125 in a constant

Reply:
also there's literally no reason for these static variables to be static

Reply:
> also there's literally no reason for these static variables to be static

the code won't compile otherwise?

Reply:
wait what
why
i don't see them being used in any other methods

Reply:
its being used in the main method, so you need it to be static

Reply:
or you could just you know
put it right in the main method

Reply:
yeah thats an option